dist/calls/calls.js

```diff
@@ -249,6 +253,27 @@
                 _fillLoader(bool, methodName);
             }
 
+            function playAudio() {
+                try {
+                        mySound = new $window.Audio(soundSrc);
+                        mySound.addEventListener('ended', function() {
```

Comment:
no need to check if Audio is supported ? I know that not all browsers supports. Maybe check and $log.error if an error